examples/servant-elm-street-main/.gitignore

```diff
@@ -0,0 +1 @@
+.stack-work/
```

Comment:
This file gets autogenerated by stack, you can delet it from the example

Reply:
done